CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "^i[3-7]86$")
   SET(XNNPACK_TARGET_PROCESSOR "x86")
 ELSEIF(CMAKE_SYSTEM_PROCESSOR STREQUAL "AMD64")
   SET(XNNPACK_TARGET_PROCESSOR "x86_64")
-ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "^armv[5-8]")
+ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "^(armv[5-8]|arm.*)$")
```

Comment:
this is for aarch32 builds, which is supported up to armv8 but not armv9.  Can you clarify what 'arm' the processor is reporting?